src/Tools/Source/RunTests/TestRunner.cs

```diff
@@ -225,6 +225,26 @@ string MakeHelixWorkItemProject(WorkItemInfo workItemInfo)
                     command.AppendLine($"{setEnvironmentVariable} DOTNET_DbgEnableMiniDump=1");
                     command.AppendLine($"{setEnvironmentVariable} DOTNET_DbgMiniDumpType=1");
                     command.AppendLine($"{setEnvironmentVariable} DOTNET_EnableCrashReport=1");
+
+                    // Find the eng directory
+                    var startPath = Path.GetDirectoryName(typeof(TestRunner).Assembly.Location)!;
+                    var path = Path.GetDirectoryName(startPath)!;
+                    var gcPath = Path.Combine(path, "eng", "libclrgc.dynlib");
```

Comment:
Should this be `dylib` instead?

Reply:
![image](https://github.com/dotnet/roslyn/assets/146967/d6c67ace-c7b5-4ec0-8a8c-f8f6b9d7c127)

Reply:
Yeah it probably helps if I spell the file name correctly.